src/EnergyPlus/OutputProcessor.cc

```diff
@@ -8117,11 +8120,12 @@ void InitPollutionMeterReporting(EnergyPlusData &state, std::string const &Repor
     using namespace OutputProcessor;
     // SUBROUTINE PARAMETER DEFINITIONS:
     //             Now for the Pollution Meters
-    static Array1D_string const PollutionMeters({1, 29},
+    static Array1D_string const PollutionMeters({1, 30},
```

Comment:
Is 30 actually `ResourceType::Num` or `SomethingElse::Num`?

Reply:
Also, this should be a `constexpr std::array<std::string_view>`, not `Array1D_string`.